### ISSUE TITLE
Store is undefined when nesting in require (require.js)

### ISSUE THREAD

**Issue by whatsdis** [marcuswestin/store.js#91]
Having some issues where I try to load store.js through require.js and the result is that it differs from site to site (I am creating a bookmarklet). I am using the list of sites from here. https://github.com/jrburke/requirejs/wiki/Sites-using-RequireJS.

bbc.com and fool.com works, but the other ones (http://www.rtt.ag/en, http://www.usatoday.com/, http://www.hallmark.com/easter/ideas/easter-recipes/) are inconsistent. For example, sometimes the jquery is unavailable, sometimes the qtip is unavailable, sometimes store is unavailable.

```
if (window.require) {
    requirejs.config({
        paths: {
            jq: '//cdnjs.cloudflare.com/ajax/libs/jquery/1.8.2/jquery.min',
            qtip2: '//cdnjs.cloudflare.com/ajax/libs/qtip2/2.2.0/jquery.qtip',
            json2: '//cdnjs.cloudflare.com/ajax/libs/json2/20130526/json2.min',
            jstore: '//cdnjs.cloudflare.com/ajax/libs/store.js/1.3.14/store.min'
        },
        map: {
            'qtip2': { 'jq': 'jquery-private' },
            'json2': { 'jq': 'jquery-private' }
        },
        findNestedDependencies: true
    });

    define('jquery-private', ['jq'], function () {
        return jQuery.noConflict(true);
    });

    require(['jstore'], function(store){
        require(['jquery-private', 'qtip2', 'json2'], function (jay) {
            console.log(store);
            console.log(jay);
            qtip2Initializer(jay,store);

        });
    });

}

function qtip2Initializer(jay,store) {
    console.log(jay().jquery);
    // Your logic to execute when qTip2 is available
    jay(document).ready(function () {
      $('a').qtip();
      store.set("test",1);
    });
}
```

**Comment by marcuswestin** [comment]
This may very well have been solved by 7c65e0a646c9c4743. @kgust, can you please verify and/or close?

Thanks for the patience!

**Comment by marcuswestin** [comment]
This may very well have been solved by 7c65e0a646c9c4743. @whatsdis, can you please verify and/or close?

Thanks for the patience!

**Comment by marcuswestin** [comment]
Feel free to reopen with more info.
